### DISCUSSION TITLE
穹妹！我超喜欢！

### DISCUSSION THREAD

**Discussion by jiishdhhwhahysyd** [share121/inter-knot#133] [其他]
无论在什么地方，什么时候，在我们的头顶都是同样悠远的天穹，就好像是永远都无法分开的羁绊一样 ![66b6d9bda4025](https://github.com/user-attachments/assets/633d5b9d-f1ff-461d-b34a-f3cdbf8e49df)

**Comment by NANTI34** [comment]
刑啊，很可拷啊

**Comment by AL-Pinecore** [comment]
好家伙，缘神，启动！

**Reply by NANTI34** [reply]
启动！

**Comment by SheepyTRF** [comment]
马上就到你家门口

**Comment by Vrey2077** [comment]
缘神，启动!

**Comment by myt521** [comment]
我超，缘

**Comment by NOVAQUQ** [comment]
想看铃妹_(´ཀ`」 ∠)__

**Comment by Hexiaoling457** [comment]
真邢！

**Comment by cow-your-sister** [comment]
缘神，启动！

**Comment by baimo2004** [comment]
你真刑阿！

**Comment by hao123nevergiveup** [comment]
真是可惜你了不是他哥哥

**Comment by EllenJoe14** [comment]
缘神 启动！

**Comment by jiishdhhwhahysyd** [comment]
为什么在绳网看不到我的帖子了QWQ

**Comment by Mnpolize** [comment]
为什么层主帖子还有水印的！这样会造成版权纠纷，我们绳匠不希望看到这样的场面！（恼）

**Comment by EntID-Ke** [comment]
*附上一张经过本人处理的穹妹图片*[**原图**](https://krseoul.imgtbl.com/i/2024/08/10/66b6dcd3ce186.jpg)
![KasuganoSoraPicture](https://github.com/user-attachments/assets/ba65e55b-2371-4081-bd76-12805844f515 "KasuganoSora")
以及一张超分后的[**穹妹图片**](https://krseoul.imgtbl.com/i/2024/08/10/66b6d9bda4025.png)
使用esrganv3-annime模型
分辨率1076\*1920\>2152\*3840
占用大小757kb\>11.2mb

**Comment by DiamondDiavolo** [comment]
逆天

**Comment by jiishdhhwhahysyd** [comment]
十分感谢@Mnpolize @EntID911 
图片已换源
